cpp/arcticdb/version/version_core.cpp

```diff
@@ -1214,13 +1217,17 @@ VersionedItem sort_merge_impl(
                 sk->unset_segment();
             }
             aggregator.commit();
+            pipeline_context->desc_->set_sorted(previous_sorted_value.has_value() ?
```

Comment:
The double ternary operator is a bit dense, explode out to `if`? Normally I'd put this kind of state machine logic with the `SortedValue` enum itself (it's also duplicated below).

Reply:
I accidentally committed to this branch for a PR that's gonna follow, so this has gone